woocommerce/woocommerce
# Remove the "Always show pre-publish checks" from pre-publish panel

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes the "Always show pre-publish checks" option from the pre-publish panel. We should show the panel all the time and not allow users to turn it off because there isn't a way to re-enable it.

Before

![Screenshot 2024-03-18 at 11 37 20 AM](https://github.com/woocommerce/woocommerce/assets/1314156/7318bbf8-b836-45bb-8bc2-d5066c436f3c)


After

![Screenshot 2024-03-18 at 11 24 23 AM](https://github.com/woocommerce/woocommerce/assets/1314156/d93136f3-c3c2-4e07-b231-f16c6c0436a0)

Closes #45668.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the New product editor and the Prepublish sidebar (product-pre-publish-modal) are enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Add a product name and press `Publish`.
3. The "Always show pre-publish checks" checkbox in the pre-publish panel shouldn't be visible.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
